### PR TITLE
fix: at commons: 4.0.1 uptake and improved pub points [at_invitation_flutter]

### PR DESCRIPTION
**Pull request by Sardar1208**
**What I did** - updated all dependencies (including at_commons: 4.0.1) and fixed warnings, added documentation
at_commons ticket link - https://github.com/atsign-foundation/at_widgets/issues/794
pub points ticket - https://github.com/atsign-foundation/apps/issues/780

### CODE REVIEW

File: packages/at_invitation_flutter/lib/widgets/otp_dialog.dart [hunk 1]
```diff
@@ -38,7 +38,7 @@ class _OTPDialogState extends State<OTPDialog> {
   @override
   Widget build(BuildContext context) {
     SizeConfig().init(context);
-    var deviceTextFactor = MediaQuery.of(context).textScaleFactor;
+    var deviceTextFactor = MediaQuery.of(context).textScaler.scale(20) / 20;
```
Comment (gkc) [inline packages/at_invitation_flutter/lib/widgets/otp_dialog.dart:41]: what's the purpose of this change?

Comment (Sardar1208) [reply packages/at_invitation_flutter/lib/widgets/otp_dialog.dart:41]: textScaleFactor is deprecated, so updated that with textScaler
